custom-deploy-steps/create-task-definition/action.yml

```
@@ -0,0 +1,97 @@
+name: Create task definition revision
+description: Use image_uri to create new task definition revision, and deregister previous active task definition
+
+inputs:
+  application_id:
+    description: Application ID also the task definition name in this case
+    required: true
+  image_uri:
+    description: The URI of the container image to insert into the ECS task definition
+    required: true
+
+  aws_access_key_id:
+    description: AWS access key
+    required: true
+  aws_secret_access_key:
+    description: AWS secret access key
+    required: true
+  aws_region:
+    description: AWS region
+    required: true
+
+outputs:
+  previous-task-definition-arn:
+    description: "ARN of previous task definition revision"
+    value: ${{ steps.get-github-action-task-definition-arn.outputs.task-definition-arn }}
+  latest-task-definition-arn:
+    description: "ARN of new task definition revision"
+    value: ${{ steps.deploy-task-definition.outputs.task-definition-arn }}
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+
+    - name: Get latest active task definition revision created by Pulumi
+      id: get-pulumi-task-definition-arn
+      shell: bash
+      env:
+        APPLICATION_ID: ${{ inputs.application_id }}
+        AWS_REGION: ${{ inputs.aws_region }}
+      run: |
+        TASK_DEFINITION_ARN=$(
+          poetry run --directory ${{ github.action_path }} actions_helper get-active-task-definition-arn-by-tag \
+            --application-id "$APPLICATION_ID" \
+            --tags "created_by:Pulumi, Name:$APPLICATION_ID" \
```

```suggestion
            --tags "created_by:Pulumi,Name:$APPLICATION_ID" \
```
I understand it doesn't matter since we are calling .strip() on this, but this would be more consistent with the error message 

```
  "Invalid task definition tag format, expects 'key:value,key:value'",
```
from above.